scripts/apps/master-desk/MasterDesk.tsx

```diff
@@ -91,6 +93,7 @@ export class MasterDesk extends React.Component<{}, IState> {
         if (appConfig?.apps?.includes('superdesk-planning')) {
             this.setState({planning: true});
         }
+        this.setState({planning: true});
```

Comment:
When would `state.planning` be false? It looks to me, that if you are setting it to `true` unconditionally, it would never be `true` after the component has mounted. What is the purpose of a variable then?

Reply:
fixed, sorry, didn't need planning, but needed to test assignments.